packages/tests/shadydom/runner.html

```diff
@@ -26,6 +26,12 @@
     'shady-dynamic.html?forceCustomElements=true',
     'shady-dynamic.html?noPatch=true&forceCustomElements=true',
     'shady-dynamic.html?noPatch=on-demand&forceCustomElements=true',
+    'shady-dynamic.html?querySelectorImplementation=selectorEngine',
```

Comment:
Are there tests for `querySelectorImplementation=native`?

Reply:
No, I've just added those though. I had to update some of them since they were doing things that aren't supported with that option.